Setup/setup.iss

```diff
@@ -9,12 +9,13 @@
 #define App1240Id "{{63B0C2BF-F4A5-43BA-AE1A-B30C88E3E369}"
```

Comment:
you need to remove the old AppIds, and also to update the InitializeSetup method with the new App200Id